.kitchen.yml

```diff
@@ -12,29 +12,60 @@ provisioner:
   data_bags_path: test/data_bags
 
 platforms:
+  # Ubuntu 18.04 is what’s used on the live OSM servers,
+  # according to https://hardware.openstreetmap.org/
   - name: ubuntu-18.04
+  - name: ubuntu-18.04-ci
+    # Driver, transport, and provisioner using Docker and Chef Infra Client,
+    # necessary when used on virtual CI boxes that refuse to run Virtualbox or Vagrant.
+    # https://github.com/someara/kitchen-dokken
+    driver:
+      name: dokken
+      image: dokken/ubuntu-18.04
+      provision: true
+      # Docker and systemd need full capabilities granted in privileged mode
+      privileged: true
+      # Run systemd init to allow services like MySQL, etc. to function
+      pid_one_command: /bin/systemd
+      intermediate_instructions:
+        # Help APT find its required sources on first run
+        - RUN /usr/bin/apt-get update -y
+    provisioner:
+      name: dokken
+      product_name: chef
+      product_version: 14
+      data_bags_path: test/data_bags
+      # Automatically accept Chef license when running headlessly
+      chef_license: accept
```

Comment:
if you're using chef 14 there's no need to accept the license

Reply:
Yes I was going to point that out ;-) That said my first attempt at running it is trying to use 15 for some reason...

Reply:
Interesting! We added this because we saw license-related failures until we added this line.

Reply:
You're seeing that because this config isn't actually correct. product_version is for omnibus installs in kitchen-vagrant. Kitchen-dokken is fast in part because it just mounts the chef docker container onto the OS container image. Due to that you specify the chef version a bit differently. See the example here: https://github.com/someara/kitchen-dokken#usage

Reply:
@tomhughes Feel free to fill out https://www.chef.io/nonprofit-entitlement-program/ if you want to use Chef 15.

Reply:
Chef 14 fix for me: 
```
diff --git a/.kitchen.yml b/.kitchen.yml
index 449f56dc..fe3e165d 100644
--- a/.kitchen.yml
+++ b/.kitchen.yml
@@ -22,6 +22,7 @@ platforms:
     driver:
       name: dokken
       image: dokken/ubuntu-18.04
+      chef_version: 14
       provision: true
       # Docker and systemd need full capabilities granted in privileged mode
       privileged: true
@@ -33,10 +34,7 @@ platforms:
     provisioner:
       name: dokken
       product_name: chef
-      product_version: 14
       data_bags_path: test/data_bags
-      # Automatically accept Chef license when running headlessly
-      chef_license: accept
     transport:
       name: dokken
```

Reply:
@firefishy I’m curious - what problem occurs for you with these lines present? 

Certainly seems like removing them is the right thing to do, but it’s confusing to me since neither me or @migurski ran into any issues with these in place.

Reply:
@jalessio The problem is that you are actually running with chef 15 not chef 14 and although the test you chose works under chef 15 not all of them do, and chef 14 is what we use in production right not and what you should testing.

Reply:
https://github.com/tomhughes/openstreetmap-chef/commit/2744cdd0988f14d5bf2cf9c9563b3fb96d60bf61 is the configuration I am currently testing with - most of the tests are working and I am just fixing up some long standing issues that affect even our current environment.

Reply:
These are great suggestions, thanks for the detail and hints @tas50.

@tomhughes it looks like you’re constructing your own version of our branch in parallel, would you prefer to take over this PR?